app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
You need to remove `uploaded_subjects_count` from the [`attributes` list](https://github.com/zooniverse/Panoptes/pull/1357/files#diff-270dcd01883f952843b67f5bb9c5c93dL9) and the [section making sure it's only serialized when it's your own user](https://github.com/zooniverse/Panoptes/pull/1357/files#diff-270dcd01883f952843b67f5bb9c5c93dL69)